modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -20,51 +18,54 @@ To configure log forwarding to Loki, you must create a `ClusterLogForwarder` cus
 +
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
-  serviceAccountName: <service_account_name> <3>
+  serviceAccountName: 
+    name: <service_account_name> #<1>
   outputs:
-  - name: loki-insecure <4>
-    type: "loki" <5>
-    url: http://loki.insecure.com:3100 <6>
-    loki:
-      tenantKey: kubernetes.namespace_name
+  - name: loki-insecure #<2>
+    type: loki #<3>
+    loki: 
+      url: http://loki.insecure.com:3100 #<4>
       labelKeys:
-      - kubernetes.labels.foo
-  - name: loki-secure <7>
-    type: "loki"
-    url: https://loki.secure.com:3100
-    secret:
-      name: loki-secret <8>
+        - kubernetes.labels.foo
+      tenantKey: kubernetes.namespace_name
+  - name: loki-secure #<5>
+    type: loki
     loki:
-      tenantKey: kubernetes.namespace_name <9>
+      authentication:
+        token:
+        from: <secret_or_serviceAccount>
+        secret: #<6>
+          key: <secret_key>
+          name: loki-secret 
       labelKeys:
-      - kubernetes.labels.foo <10>
+        - kubernetes.labels.foo #<7>
+      tenantKey: kubernetes.namespace_name #<8>
```

Comment:
Is now a template and combination of static and dynamic values inside curly brackets (with a static fallback)

Reply:
updated